backend/internal/utils/env.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import "os"
+
+// GetEnvOrDefault retrieves the value of the environment variable named by the key.
+// If the variable is not present in the environment, then the provided fallback value is returned.
+func GetEnvOrDefault(key, fallback string) string {
+	value, exists := os.LookupEnv(key)
```

Comment:
[optional] 

Since the check is on `!exists || value == ""` maybe using `os.GetEnv` and checking only for `== ""` would be equivalent and simpler.